.travis.yml

```diff
@@ -24,7 +24,7 @@ install:
   #   /home/travis/gopath/src/github.com/harmony-one/harmony
   # https://docs.travis-ci.com/user/languages/go/#go-import-path
   - echo $TRAVIS_PULL_REQUEST_BRANCH
-  - TEST_REPO_BRANCH="master"
+  - TEST_REPO_BRANCH="feature/effective-gas-price"
```

Comment:
not sure we want this to be be in the repo. @mur-me what do you think ?

Reply:
hey, @sophoah, per my idea that I have in my mind, this line should be chanded when you want to test you changes in both repos.

So here is should returned back to master and simultaneously PR to the test should be merged, because we have chicken vs egg problem, because changes are in both repos

Reply:
https://github.com/harmony-one/harmony/pull/4828